repositories.bzl

```diff
@@ -127,8 +127,8 @@ def _rules_proto_repositories():
         urls = ["https://github.com/bazelbuild/rules_proto/archive/" + RULES_PROTO_COMMIT + ".tar.gz"],
     )
 
-ZLIB_RELEASE = "1.2.11"
-ZLIB_SHA256 = "c3e5e9fdd5004dcb542feda5ee4f0ff0744628baf8ed2dd5d66f8ca1197cb1a1"
+ZLIB_RELEASE = "1.2.12"
```

Comment:
Is this related?

Reply:
No it's not directly related to the fix. However, the zlib 1.2.11 archive is no longer available for download. I had to update it to 1.2.12 so I could build & test.